### DISCUSSION TITLE
What can we exactly overwrite in a breakpoint ?

### DISCUSSION THREAD

**Discussion by GuyoST** [nolimits4web/swiper#5546] [Q&A] [answered]
Hello every one, 

I just discovered swiper and it's awesome.
I made a lot tests with it today I have a question about breakpoints.

I could see we can provide a `SwiperOptions `per breakpoint, but when I tried it, it seems that it's not working for all values.
For example, I have swiper with 1 image per slide and slide effect.

I add this breakpoints : 
```
1000: {
    effect: "fade",
    slidesPerView: 2
  }
```

So when my screen is larger than 1000 pixels, I get 2 images per view correctly, but the effect is still `slide`, not `fade`
So I guess we can't overwrite everything, but is there a list of what we can overwrite ?

Thank you :)

**Comment by Arhell** [comment, marked as answer]
https://swiperjs.com/swiper-api#param-breakpoints

> Such parameters like loop and effect won't work

**Reply by GuyoST** [reply]
Thank you ! Sorry that I missed this page :)
